pkg/controller/telemetry/metric.go

```diff
@@ -203,9 +203,15 @@ func buildMetricFromWorkload(dstWorkload, srcWorkload *workloadapi.Workload) com
 	}
 
 	trafficLabels := commonTrafficLabels{}
+	services := dstWorkload.Services
+	svcHost := ""
+	for k := range services {
+		svcHost = strings.Split(k, "/")[1]
```

Comment:
Maybe we should defensively check of the length of the result returned by `strings.Split` to prevent out-of-bounds access.